jsettlers.logic/src/jsettlers/logic/movable/Movable.java

```diff
@@ -78,7 +78,7 @@
 
 	private ShortPoint2D position;
 
-	private ShortPoint2D moveToRequest = null;
+	private ShortPoint2D targetPosition = null;
```

Comment:
moveToRequest is intended to be the position the user wants the movable to move to.